docs_user/modules/con_bare-metal-provisioning-service-configurations.adoc

```diff
@@ -1,18 +1,16 @@
 [id="con_bare-metal-provisioning-service-configurations_{context}"]
 
-= Bare Metal Provisioning service configurations
+= {bare_metal} configurations
 
```

Comment:
I have general questions about this entire section, "{bare_metal} configurations." 

1. How much of this content do downstream customers need to know in order to adopt their Bare Metal service to the RHOSO environment? 
2. Can we select what is important for downstream customers to know, and then either remove or conditionalize the rest for upstream only?